test/runtests.jl

```diff
@@ -4,7 +4,8 @@ using Test
 @testset "TBLogger" begin
     include("test_TBLogger.jl")
 end
-
+Pkg.add("Flux")
+Pkg.add("Metalhead")
```

Comment:
Install packages required for tests using the Project.toml.
https://docs.julialang.org/en/v1.0/stdlib/Pkg/#Test-specific-dependencies-1